<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>677</width>
    <height>739</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>RoadRoller</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <property name="sizeConstraint">
       <enum>QLayout::SetMinimumSize</enum>
      </property>
      <item>
       <layout class="QHBoxLayout" stretch="0,0,0,0,0,0">
        <property name="spacing">
         <number>20</number>
        </property>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3"/>
        </item>
        <item>
         <widget class="QPushButton" name="loadButton">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Maximum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="text">
           <string>Load Image...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonSave">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Save...</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Quality factor:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="sliderQuality">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Minimum">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="sliderPosition">
           <number>90</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="labelQualityValue">
          <property name="minimumSize">
           <size>
            <width>40</width>
            <height>0</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true"/>
          </property>
          <property name="text">
           <string>90</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QLabel" name="labelOriginalTitle">
        <property name="text">
         <string>&lt;h3&gt;Original&lt;/h3&gt;</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelCompressedTitle">
        <property name="text">
         <string>&lt;h3&gt;Compressed&lt;/h3&gt;</string>
        </property>
        <property name="textFormat">
         <enum>Qt::AutoText</enum>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QScrollArea" name="scrollOriginal">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="layoutOriginal">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>319</width>
           <height>580</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QScrollArea" name="scrollCompressed">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="layout2">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>318</width>
           <height>580</height>
          </rect>
         </property>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>677</width>
     <height>24</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
